includes/gateways/class-api-gateway.php

```diff
@@ -524,45 +513,54 @@ private function is_poll_response_valid( $response ) {
 	 * @return bool|\WP_Error
 	 */
 	public function get_is_user_verified() {
-		$response = $this->perform_request( 'GET', '/account/verified' );
-		if ( is_wp_error( $response ) ) {
-			return $response;
-		}
+		$response_data = $this->get_account_info();
 
-		$body          = wp_remote_retrieve_body( $response );
-		$response_data = json_decode( $body, true );
-
-		if ( null === $response_data || ! is_array( $response_data ) ) {
-			if ( isset( $response_data['error'] ) ) {
-				return new \WP_Error( $response_data['error'], $response_data );
-			}
-			return new \WP_Error( 'decode-failed' );
-		}
-
-		return $response_data['is_verified'];
+		return isset( $response_data['is_verified'] )
+			? (bool) $response_data['is_verified']
```

Comment:
How about `return (bool) $response_data['is_verified'] ?? false;` ?

Reply:
I have a warning here due to coalescent operator not being available on PHP <5.6, not sure this is a thing, but maybe just keep it as it is for now.

Reply:
Oh I totally forgot about WP's minimum supported version. My bad, sorry.